test/segment-loader.test.js

```diff
@@ -403,7 +403,7 @@ QUnit.test('only appends one segment at a time', function(assert) {
   assert.equal(loader.mediaRequests, 1, '1 request');
 });
 
-QUnit.test('adjusts the playlist offset if no buffering progress is made', function(assert) {
+QUnit.skip('adjusts the playlist offset if no buffering progress is made', function(assert) {
```

Comment:
This test is actually outdated from before simple-fetcher times when there was the offset correction variable to handle incorrect guesses, (now we just walk forward), it just wasnt skipped/removed previously because it happened to pass after segment loader changes. If you notice the other playlist offset tests are also skipped. I was planning on cleaning up skipped tests when the loaders get refactored